### DISCUSSION TITLE
Smart card not supported for Account

### DISCUSSION THREAD

**Discussion by paracetamol32** [gravitational/teleport#42575] [Q&A]
653 / 5 000
hello,
I had a teleport setup that worked very well with access to all my windows machines.
I was forced to migrate the PKI to a new server. I did as recommended by MS a backup of the pki and the registry. I installed and restored the new server.
Now that I try to go to a Windows server I get a message telling me that Smart card not supported for Account.
I activated the kerberos event on Windows and I got the following error:
The Kerberos Key Distribution Center (KDC) for the domain corp.FR does not have a certificate installed or does not support logon using certificates: 0xC0000320

Have you an idea ??

**Comment by zmb3** [comment]
See also https://github.com/gravitational/teleport/discussions/33108

Your KDC certs are likely missing the smart card logon EKU.
